File: Example/NXNavigationExtensionDemo/Feature/Basic/Controllers/ViewController06_ShadowImage.m
//
//  ViewController06_ShadowImage.m
//  NXNavigationExtensionDemo
//
//  Created by Leo Lee on 2020/10/26.
//

#import <NXNavigationExtension/NXNavigationExtension.h>

#import "ViewController06_ShadowImage.h"

@interface ViewController06_ShadowImage ()

@end

@implementation ViewController06_ShadowImage

- (void)viewDidLoad {
    [super viewDidLoad];
    
}

- (UIImage *)nx_shadowImage {
    return [UIImage imageNamed:@"NavigationBarShadowImage"];
}

@end

File: Example/NXNavigationExtensionSwiftDemo/Basic/ViewController06_ShadowImage.swift
//
//  ViewController06_ShadowImage.swift
//  NXNavigationExtensionSwiftDemo
//
//  Created by Leo Lee on 2020/11/23.
//

import UIKit

class ViewController06_ShadowImage: BaseViewController {

    override func viewDidLoad() {
        super.viewDidLoad()

        // Do any additional setup after loading the view.
    }

    override var nx_shadowImage: UIImage? {
        return UIImage(named: "NavigationBarShadowImage")
    }

}
